### DISCUSSION TITLE
Looking up qualities on V:\plot-k32-2021-06-06-04-55-f2011be3f2a8bddcc2c7a785c4a148234941fd032e593c3fb704c94c8200df62.plot took: 8.77606987953186. This should be below 5 seconds to minimize risk of losing rewards.

### DISCUSSION THREAD

**Discussion by jamescjw** [Chia-Network/chia-blockchain#7033] [Support]
Warning such as below frequently:
Looking up qualities on V:\plot-k32-2021-06-06-04-55-f2011be3f2a8bddcc2c7a785c4a148234941fd032e593c3fb704c94c8200df62.plot took: 8.77606987953186. This should be below 5 seconds to minimize risk of losing rewards.

Harddisk's status is good and the harddisk temperture also low...

how to reslove this warning?Any one knows?! Thank you!!

**Comment by joelrb** [comment]
> Warning such as below frequently:
> Looking up qualities on V:\plot-k32-2021-06-06-04-55-f2011be3f2a8bddcc2c7a785c4a148234941fd032e593c3fb704c94c8200df62.plot took: 8.77606987953186. This should be below 5 seconds to minimize risk of losing rewards.
> 
> Harddisk's status is good and the harddisk temperture also low...
> 
> how to reslove this warning?Any one knows?! Thank you!!

Make sure your disk is always "awake" or is awaken every 1 or 2 seconds. Some disk "sleeps" to conserve power. So you need to find a way to keep alive your disk. 

In Mac I use the Amphitamine app ... that keeps the drives alive every 2 seconds.

**Comment by BrandtH22** [comment]
Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
